src/InstantiableEntity.php

```diff
@@ -8,8 +8,12 @@
 
 class InstantiableEntity extends BaseEntity {
 
-	function __construct( $name, array $properties = [] ) {
-		$this->name = $name;
+	protected $name;
+
+	function __construct( $name = null, array $properties = [] ) {
+		if ( isset( $name ) ) {
```

Comment:
This is approved, but do want to discuss further.

I should read dictionary entries more often! I don't even see "irrelevant" used as a synonym for moot, but I would always use it for that purpose, which I suppose is moot for this particular conversation.